### PR TITLE
Add ACL system to Keeper

### PR DESCRIPTION
**Pull request by alesapin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- New Feature

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Now clickhouse-keeper supports ZooKeeper-like `digest` ACLs.

### CODE REVIEW

File: src/Common/ZooKeeper/IKeeper.h [hunk 1]
```diff
@@ -148,6 +148,40 @@ struct WatchResponse : virtual Response
 
 using WatchCallback = std::function<void(const WatchResponse &)>;
 
+struct SetACLRequest : virtual Request
```
Comment (alesapin) [inline src/Common/ZooKeeper/IKeeper.h:151]: Currently unused, and I'm not sure, that I'm going to implement them.

**Comment by alesapin** [comment]
???

**Comment by alesapin** [comment]
@Mergifyio update

**Comment by alesapin** [comment]
@Mergifyio update

**Comment by alesapin** [comment]
` Functional stateless tests (release, DatabaseReplicated) — fail: 20, passed: 1651, skipped: 45 ` not related to changes.

**Comment by alesapin** [comment]
No related failures, but let's check one more time.

**Comment by alesapin** [comment]
@Mergifyio update

**Comment by alesapin** [comment]
One more time and merge.

**Comment by alesapin** [comment]
Oh, no. Need some improvements.

**Comment by alesapin** [comment]
No related failures, I'll merge.

**Comment by gyuton** [comment]
Internal documentation ticket: DOCSUP-13992.

**Pull request merged**